src/foundry/client/pixi/board.d.ts

```diff
@@ -565,6 +617,21 @@ declare global {
      */
     static clearContainer(displayObject: PIXI.DisplayObject, destroy?: boolean): void;
 
+    /**
+     * Get a texture with the required configuration and clear color.
+     */
+    static getRenderTexture(options?: {
+      /**
+       * The clear color to use for this texture. Transparent by default.
+       */
+      clearColor?: number[];
+
+      /**
+       * The render texture configuration.
+       */
+      textureConfiguration?: Parameters<(typeof PIXI.RenderTexture)["create"]>[0];
+    }): PIXI.RenderTexture;
```

Comment:
Consider `InexactPartial` here. I don't think the repo as a whole is good at this but I'll try to note it wherever it crops up in this, I'll add an issue to apply `InexactPartial` for every optional property elsewhere in the repo.